src/panels/lovelace/editor/view-editor/hui-edit-view.ts

```diff
@@ -87,6 +87,16 @@ export class HuiEditView extends LitElement {
     return this.shadowRoot!.querySelector("ha-paper-dialog")!;
   }
 
+  private get _viewConfigTitle(): string {
+    if (!this._config || !this._config.title || this._config.title === "") {
+      return this.hass!.localize(
+        "ui.panel.lovelace.editor.edit_view.header"
+      );;
+    }
+
+    return this.hass!.localize("ui.panel.lovelace.editor.edit_view.header_name", this._config.title)
```

Comment:
Does the key `header_name` exist? You have to add it to `src/translations/en.json`: `Edit view {name}`
And you have to pass the name of the placeholder you want to replace:
```suggestion
    return this.hass!.localize("ui.panel.lovelace.editor.edit_view.header_name", name, this._config.title)
```